operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -76,6 +76,7 @@
     public static final String CA_KEY = "ca.key";
     public static final String CA_CRT = "ca.crt";
     public static final String IO_STRIMZI = "io.strimzi";
+    public static final String ANNO_STRIMZI_IO_FORCE_RENEW = "strimzi.io/force-renew";
```

Comment:
Not for this PR but we should try to centralise these annotations (we have for pod deletion, rolling updates)